test/hotspot/jtreg/compiler/arguments/TestC1Globals.java

```diff
@@ -56,25 +36,23 @@
  * @bug 8318817
  * @requires vm.debug
  * @requires os.family == "linux"
- * @summary Test flag with large value combined with transparent huge pages on
+ * @summary Test flag with max value combined with transparent huge pages on
  *          Linux.
  *
- * @run main/othervm -XX:NMethodSizeLimit=351658240
```

Comment:
What were these large sizes of NMethodSizeLimit meant to test?  Removing these test cases because of a problem with aarch64 seems wrong, unless these test cases really have no value for other platforms.

Reply:
> What were these large sizes of NMethodSizeLimit meant to test? Removing these test cases because of a problem with aarch64 seems wrong, unless these test cases really have no value for other platforms.

That would be my guess.